package.json

```diff
@@ -45,13 +45,11 @@
     "broadcastchannel-polyfill": "^1.0.0",
     "dedent": "^0.7.0",
     "history": "^4.9.0",
-    "marksy": "^8.0.0",
```

Comment:
Did not find any usages of this lib in the codebase.  Does it seem safe to remove?  (Removing dependencies is hard without full context!)

Reply:
After investigating the git history, looks like this lib was used for an old Markdown component.  
So yes, safe to delete now!